back/boxtribute_server/graph_ql/definitions/protected/types.graphql

```diff
@@ -46,6 +46,7 @@ type Box implements ItemsCollection {
 }
 
 type HistoryEntry {
+  " IDs are not guaranteed to be unique due to history data being pulled from multiple sources "
```

Comment:
please make a consolidation task out of changing this comment: Rather add a prfix to the ID

Reply:
see [653b0ad](https://github.com/boxwise/boxtribute/pull/1472/commits/653b0ad871ec0fb64765f7983b0b5d85ee51ccd5)